package.json

```diff
@@ -440,7 +453,8 @@
                     "default": {
                       "pipeCwd": "${workspaceRoot}",
                       "pipeProgram": "enter the fully qualified path for the pipe program name, for example 'c:\\tools\\plink.exe'",
-                      "pipeArgs": []
+                      "pipeArgs": [],
+                      "debuggerPath" : "Please enter the debugger path here, for example c:\\clrdbg\\clrdbg"
```

Comment:
Same suggestion as in attach: I think we don't want the OS-specific 'debuggerPath' properties.